src/grammar_parser/cfgparser.py

```diff
@@ -85,7 +91,7 @@ class bcolors:
     UNDERLINE = '\033[4m'
 
 
-class Option:
+class Option(object):
```

Comment:
Don't need this.

Reply:
But, why not? Do you want me to remove it?

Reply:
The inheritance from object is default in python3. This syntax is only used for backwards compatibility with python2.

Reply:
So this is an improvement. So that it behaves the same in python2 and python3

Reply:
But we are moving to python3, lets not support python2 anymore. Its a hassle maintaining a deprecated version.

Reply:
I prefer compatibility for both for a moment. But for this one we can remove it.